third_party/move/move-core/types/src/identifier.rs

```diff
@@ -89,7 +89,7 @@ pub(crate) static ALLOWED_NO_SELF_IDENTIFIERS: &str =
 ///
 /// For more details, see the module level documentation.
 #[derive(Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize, Deserialize)]
-#[cfg_attr(any(test, feature = "fuzzing"), derive(arbitrary::Arbitrary))]
+#[cfg_attr(feature = "fuzzing", derive(arbitrary::Arbitrary,dearbitrary::Dearbitrary))]
```

Comment:
the `any(test, feature = "fuzzing")` here probably existed for a reason. maybe we should try to keep it that way?
same for couple of other places.